service/src/main/java/uk/gov/hmcts/reform/fpl/validation/interfaces/time/TimeDifference.java

```diff
@@ -1,8 +1,10 @@
 package uk.gov.hmcts.reform.fpl.validation.interfaces.time;
 
+import java.time.LocalDateTime;
 import java.time.temporal.ChronoUnit;
 
 public @interface TimeDifference {
+    LocalDateTime startDate = LocalDateTime.now();
```

Comment:
not sure what is purpose of it? Looks that not used

Reply:
removed 👍